lib/src/modules/thumb/Thumb.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { node, string } from 'prop-types';
+import { useImage } from 'react-image';
+import { createClassNames } from 'helpers/createClassNames';
+import Icon from 'lib/Icon';
+
+function Thumb({ src, altText, title, fallback, children, ...props }) {
+  const image = useImage({
+    srcList: src,
+    useSuspense: false
+  });
+
+  const [
+    minimumLoadingTimeSurpassed,
+    setMinimumLoadingTimeSurpassed
+  ] = useState(true);
+  useEffect(() => {
+    setTimeout(() => setMinimumLoadingTimeSurpassed(true), 250);
+  }, []);
```

Comment:
When using the module, it helps if the user has a consistent experience so;

- the user sees a loader (something must be coming)
- the user then sees image appear after it's loaded
- win 🥳 

Without this we get some instances where the image has "loaded" but it takes X time to render on the page which gives us strange whitespace which causes the UI to look odd.

After some testing especially in a collection of thumbs, this is even more apparent and this minimal wait time really helps to give us that consistent experience with loading images 😄

---

Saying all of that! This was implemented when the loaded was unmounted which isn't a thing SOOOOO it might not be needed. I'll re-test this and it's very possible that it isn't required anymore.

I'll update and resolve if I remove it 👍

Reply:
This has been removed, this implementation is a hack to try and get around the issue of us overusing inline SVGs especially in the case where said SVG is animated.